docs/src/main/tut/datatypes/averaged_value.md

```diff
@@ -8,8 +8,16 @@ scaladoc: "#com.twitter.algebird.AveragedValue"
 
 # Averaged Value
```

Comment:
We should mention the two main facts: 1) allows you to get the average and the count with one pass over the data. 2) tracks the average using a stable algorithm that won't overflow for big data (as long as there are fewer than 2^63 items generally).